### DISCUSSION TITLE
Support for Swagger

### DISCUSSION THREAD

**Discussion by bitstan-earthyfrodo** [retypeapp/retype#338] [Q&A]
I would like to embed/implement Swagger request calls into our documentation like as GitBook supports.
Is there an existing/sexy way to implement this as well into Retype docs?

**Example from GitBook**
![Swagger](https://user-images.githubusercontent.com/102163821/173418304-199d30dc-8cc3-438f-bcd1-7cd77d08441d.JPG)

**Result in docs**
![Swagger_1](https://user-images.githubusercontent.com/102163821/173418287-2aad978b-0a21-446a-b74b-09fd82c294a7.JPG)

**Comment by geoffreymcgill** [comment]
Cool feature, although Retype does not have an equivalent. It is certainly something we can investigate though.

The basic formatting could be replicated with Retype components, although manually.
